### ISSUE TITLE
Sprint 2 – Sprint Planning (5-12-2016)

### ISSUE THREAD

**Issue by huylin25** [mewind/RSDSupermarketPOS#5]
Hi guys, today we are going to start our Sprint 2. I'm the scrum master of this Sprint.
In this sprint we will mainly focus on Product module, do you guys have any idea on this module?

**Comment by mewind** [comment]
I've already read some of the user stories from the user , in the product module, the user hopes to search the products in different categories, so I think for the design of the module we can first separate them into 3 different categories which is Stationary, Clothing and Groceries for the ease of searching product.

**Comment by juzplayplay** [comment]
Sin Nee you're right, I think we can also include search function for the users who exactly know what they want so they can search the product easily

**Comment by huylin25** [comment]
Ya, that's also included in the user stories too. So you guys hope to provide a better searching experience by categorizing the items into 3 different categories and allow users to search the particular items by typing in the name of the item?

**Comment by juzplayplay** [comment]
Yes, That is what I'm thinking.

**Comment by lkc442012** [comment]
I agree with both Raymond and SinNee that this categorization and search function will allow the users to search the product they want more easily. So do you guys think Add To Cart should be implemented in a different form ?

**Comment by huylin25** [comment]
Err... In my opinion I think the way to perform add to cart should be double clicking the product we want in the product list, and there is a Add To Cart form popup on the window, then we can enter the quantity we want for the product then click confirm button to add them into cart...

**Comment by mewind** [comment]
Ya, I actually agree with HuyLin which I think the same Add To Cart form will be pop up when the user choose the product in any category, I think we do not need a separate list for the Cart in the application, the user can always check what is in their cart in the Payment Tab straight away, that might able to make the system more simple to use.

**Comment by lkc442012** [comment]
Sounds a good idea Sin Nee! That means we do not need to develop a redundant function which is the Add to Cart list, all we need to do is just have a Add to Cart window pop up and straight show the selected products and price in the Payment Tab, this will also speed up our development in Payment tab later

**Comment by huylin25** [comment]
Okay, so tentatively we follow this logic when developing Add to Cart function which is the most tricky part in this module in my opinion. Anyway, I think at this moment we need not to discuss much on the logic part because I think it will alter after we start developing the module. Haha.

**Comment by juzplayplay** [comment]
HuyLin point out the truth, but anyway I think the logic is quite correct, we can have a try first and discuss the difficulties later.

**Comment by mewind** [comment]
Okay no problem. Oh ya I think we will need to check the stock quantity before the user can add the item into cart and show them some error messages if the product is out of stock.

**Comment by huylin25** [comment]
Yes, that is also crucial part in the module. Okay, since we've already come up with the basic ideas to develop the module, may I know who is able to in charge of this module?

**Comment by mewind** [comment]
I think I'll take the categorization part and the search function ?

**Comment by mewind** [comment]
I think I'll take the Add To Cart function and Search function?

**Comment by huylin25** [comment]
Okay no problem. So Raymond, can you do mainly the design part which include the categorization of the products and check database for stock quantities ?

**Comment by juzplayplay** [comment]
Sure,I'm okay with it.

**Comment by huylin25** [comment]
Okay, KarCheng will assist both of you when during the development. So by the day finishing this Sprint we need to complete the Payment function include at least with: Categorization design,  Add to Cart function, Check stock quantity and Search Product function. Did I missed anything?

**Comment by mewind** [comment]
I think no?

**Comment by huylin25** [comment]
Okay, then we will stop here. See you guys again tomorrow, please don't hesitate to find me if you guys have any problem.
